Frontend-Development/README.md

```diff
@@ -56,6 +56,9 @@
 - [Learn JavaScript - Codecademy](https://www.codecademy.com/learn/introduction-to-javascript)
 - [Free JS Courses for Beginners - FreeCodeCamp](https://www.freecodecamp.org/news/learn-javascript-free-js-courses-for-beginners/)
 
+### Understanding JavaScript
+- [How JavaScript Works- Akshay Saini(Youtube)](https://www.youtube.com/playlist?list=PLlasXeu85E9cQ32gLCvAvr9vNaUccPVNP)
```

Comment:
Please mention that this is the Namaste JS playlist. You can name it `NamasteJS/How JS works - Akshay Saini`.

Also please remove the extra line you have added